content/posts/why-sre/index.mdx

```diff
@@ -0,0 +1,163 @@
+ <!-- Title, author, date block standard in all blog post files -->
+# Why SRE?
+*Author(s):@david-martin*<br/>
+<!-- Hide this human-readable detail so content doesn't appear to be stale or dated
+*Date Published: 2022-05-16*<br/>
+*Date Updated: N/A or YYYY-MM-DD*<br/>
+-->
+
+<!--
+Categories:
+Tags:
+-->
+
+There are lots of reasons you may be thinking about Site Reliability Engineering (SRE), both the role (Engineer) and the concept (Engineering).
+The role can have a very specific meaning in your organisation, which is fine.
+In fact it needs to be specific so you can hire accordingly, onboard and train people to do a specific job.
+It's your implementation of the SRE principles.
+
+The SRE concept and principles are born from Google and their [various publications][1].
+Principles such as *Embracing Risk*, *Service Level Objectives* and *Eliminating Toil*.
+Following on from the principles are practices like *Managing Incidents*, a *Postmortem Culture* and *Testing for Reliability*.
+Here, we’ll present some case studies in Red Hat where SRE is being practised and try to understand the reasons for choosing SRE and how it was applied.
+
+## Case Study 1: OpenShift Dedicated
```

Comment:
call this Managed OpenShift (since we also have ROSA and ARO which are all supported by SRE.